[32mINFO    [0m conftest:conftest.py:12 Initializing UserApiClient fixture
[32mINFO    [0m api_clients.users_api_client:users_api_client.py:11 UserApiClient initialized.
[32mINFO    [0m conftest:conftest.py:18 Getting token for user: test2@fake.com
[32mINFO    [0m api_clients.users_api_client:users_api_client.py:15 Trying to log in with email: test2@fake.com
[32mINFO    [0m api_clients.users_api_client:users_api_client.py:53 Login successful. Token received.
[32mINFO    [0m conftest:conftest.py:22 Token received successfully
[32mINFO    [0m conftest:conftest.py:30 Creating ContactsApiClient instance with the retrieved token